src/graphic/shape/Sector.ts

```diff
@@ -9,8 +9,20 @@ export class SectorShape {
     startAngle = 0
     endAngle = Math.PI * 2
     clockwise = true
-    cornerRadius = 0
-    innerCornerRadius = 0
```

Comment:
Do we need to compat with `innerCornerRadius` in order versions?

Reply:
The API doesn't change in ECharts, I'm not sure if zrender also needs the backward compatibility.

Reply:
I think zrender can do this kind of breaking change to keep the simplicity

Reply:
I think zrender can do this kind of breaking change to keep the simplicity